templates/app-long-text.html

```diff
@@ -428,20 +429,32 @@ <h1>Techtonica Application</h1>
     padding: 2rem;
   }
 
-  .application-form .learn-more-wrapper .learn-more-title,
-  .application-form .learn-more-wrapper .toggle-icon {
+  .application-form .learn-more-wrapper {
+    display: inline;
+  }
+
+  .application-form .learn-more-content {
+    margin: 0.5em 0 0 0;
+    font-size: large;
+  }
+
+  .application-form .learn-more-wrapper button {
+    display: inline;
+    background-color: white;
     color: #0093b5;
-    font-size: 18px;
+    font-size: large;
+    cursor: pointer;
+    padding: 0;
+    margin: 0 0 0 0.5em;
   }
 
-  .application-form .learn-more-wrapper .learn-more-content {
-    font-size: 18px;
-    color: black;
+  .application-form .learn-more-wrapper button:hover {
+    background-color: white;
+    color: #16c1f3;
```

Comment:
Can you please change all color codes to their corresponding sass variables in `_settings.scss`?
```suggestion
    color: $light-blue;
```